lib/ssh.js

```diff
@@ -5250,16 +5257,35 @@ function tryComputeSecret(dh, e) {
 }
 
 function convertSignature(signature, keyType) {
+  switch (keyType) {
+    case 'ssh-rsa-cert-v01@openssh.com':
+    case 'ssh-dss-cert-v01@openssh.com':
+    case 'ecdsa-sha2-nistp256':
```

Comment:
I'd suggest this entire `switch` statement is unnecessary and you could just perform the `replace` blind. But this particular `case` statement seems redundant as it doesn't end in the appropriate string to qualify for the replace call?

Reply:
In fact the two cases below are also identical. These should've been `ecdsa-sha2-nistp256-cert-v01@openssh.com`, `ecdsa-sha2-nistp384-cert-v01@openssh.com`, and `ecdsa-sha2-nistp521-cert-v01@openssh.com`.

Not going to fix it, because this PR realistically is not going to be merged due to the planned rewrite of this library.

Reply:
> Not going to fix it, because this PR realistically is not going to be merged due to the planned rewrite of this library.

Are this issues the same in https://github.com/mscdex/ssh2/pull/808 (which also "primarily awaits an updated PR")?